packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -0,0 +1,52 @@
+const { Command } = require('@oclif/command');
+
+const { startSpinner, endSpinner } = require('../utils/display');
+
+const inquirer = require('inquirer');
+
+class ZapierBaseCommand extends Command {
+  run() {
+    this.parseFlags();
+    return this.perform();
+  }
+
+  parseFlags() {
```

Comment:
We can if you'd like. Given that commands won't actually call it directly (it'll only be written in the declaration and in the base `.run`), I don't think it matters a ton what we call it. I'd lean on not blocking the actual `Command.parse` just in case we need it for some reason down the line, but I'm not sure it'll matter. No strong feelings from me here. 

Can you talk about your reasoning a little?

Reply:
I was thinking it can be confusing to have two methods of different names doing almost the same thing. But you're right: since `parseFlags` is going to be used by the base command only, it won't matter that much. 👍

Reply:
if you'd find it helpful, I can rename it `_parseFlags` and comment it as "private"?

Reply:
Sure, `_parseFlags` would make it more clear that it's internal.